readthedocs.yml

```diff
@@ -1,4 +1,4 @@
 ---
 version: 2
 conda:
-  environment: docs/conda.yml
+  environment: docs/environment.yml
```

Comment:
Will need to sync this change with #941

Reply:
We could also switch the dev docs to use `mamba env create -f=conda.yml` if switching the name back is too disruptive.

Reply:
Not a big deal, I think consistency is more important. Looking at a few other docs projects:

- docs.nextstrain.org: [environment.yml](https://github.com/nextstrain/docs.nextstrain.org/blob/master/environment.yml)
- Auspice: [docs/environment.yml](https://github.com/nextstrain/auspice/blob/master/docs/environment.yml)
- Augur: I can't find one?

Based on this, I'm on board with the move from conda.yml to environment.yml.

Reply:
If I can push back a little bit, I think `conda.yml` is better than `environment.yml`, as it specifies what software the file is for... not a huge deal, but to me `environment.yml` is like naming a file `data.tsv`. ;-)

Reply:
@tsibley I buy that point, would just want to see the other docs projects adopt `conda.yml` as well. I can PR those if it's what we settle on here.

Reply:
Moved back to `conda.yml`.